src/reference/03-Developers-Guide/06-Datatype/00.md

```diff
@@ -0,0 +1,494 @@
+---
+out: Datatype.html
+---
+
+Datatype
+--------
+
+datatype is a code generation library and an sbt autoplugin that generates
+growable datatypes and helps developers avoid breakage of binary compatibility.
```

Comment:
We should expand on what we mean by "growable," contrasting the datatypes generated by Datatype with plain case classes.